$NetBSD: patch-core_CMakeLists.txt,v 1.2 2021/02/01 09:08:43 kardel Exp $

	Setup NetBSD parameters
	Due to current lack of XDR quad marshalling NDMP
	needs to be disabled.

--- core/CMakeLists.txt.orig	2020-12-16 07:46:16.000000000 +0000
+++ core/CMakeLists.txt
@@ -259,10 +259,19 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD
   link_libraries(intl)
 endif()
 
+if (${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
+   set(HAVE_NETBSD_OS 1)
+   set(ndmp 0)
+   set(PSCMD "ps -ax -o pid,command")
+   include_directories(${PREFIX}/include)
+   link_directories(${PREFIX}/lib)
+   link_libraries(intl)
+endif()
+
 if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
   set(HAVE_DARWIN_OS 1)
   # set(CMAKE_INCLUDE_PATH  "${OPENSSL_INCLUDE_DIR}:${CMAKE_INCLUDE_PATH}")
-  set(CMAKE_INCLUDE_PATH "/usr/local/opt/:${CMAKE_INCLUDE_PATH}")
+  set(CMAKE_INCLUDE_PATH "${PREFIX}/opt/:${CMAKE_INCLUDE_PATH}")
   link_directories(/usr/local/lib)
 
   include_directories(/usr/local/opt/gettext/include)
@@ -648,6 +657,7 @@ try_compile(
   ${PROJECT_SOURCE_DIR}/src/compile_tests/trivially_copyable.cc
 )
 
+set(HAVE_NDMP 0)
 set(ndmp "${ndmp}")
 set(lmdb "${lmdb}")
 set(build_client_only "${client-only}")
@@ -662,7 +672,8 @@ if(${acl})
       set(HAVE_ACL 1)
       message(STATUS "acl found, libs: ${ACL_LIBS}")
     else()
-      message(FATAL_ERROR "build with acl requested, but lib not found")
+      # message(FATAL_ERROR "build with acl requested, but lib not found")
+      set(acl OFF)
     endif()
   endif()
 endif()
